index.html

```diff
@@ -151,18 +151,22 @@
     </p>
 
     <p>
-<a>Decentralized identifiers</a> (DIDs) are a new type of identifier to
-provide verifiable, decentralized digital identity. These new identifiers are
-designed to enable the controller of a <a>DID</a> to prove control over
-it and to be implemented independently of any centralized registry, identity
-provider, or certificate authority. <a>DIDs</a> are URLs that relate a
-<a>DID subject</a> to a <a>DID document</a> allowing trustable interactions with
-that subject. <a>DID documents</a> are simple documents describing how to use
-that specific <a>DID</a>. Each <a>DID document</a> can express cryptographic
-material, verification methods, or <a>service endpoints</a>, which provide a
-set of mechanisms enabling a <a>DID controller</a> to prove control of the
-<a>DID</a>. <a>Service endpoints</a> enable trusted interactions with the
-<a>DID subject</a>.
+<a>Decentralized identifiers</a> (DIDs) are a new type of identifier to 
+provide verifiable, decentralized digital identity. A <a>DID</a> may identify 
+any subject (e.g., a person, organisation, thing, datamodel, abstract entity, etc.)
+that the controller of that <a>DID</a> decides that it 
+identifies. These new identifiers are designed to enable the controller 
+of a <a>DID</a> to prove control over it and to be implemented independently 
+of any centralized registry, identity provider, or certificate authority. 
+<a>DID</a>s are URLs that associate a <a>DID subject</a> to a 
+<a>DID document</a> allowing trustable interactions associated with that subject. 
+<a>DID document</a>s describe how to use that specific <a>DID</a>. Each 
+<a>DID document</a> can express cryptographic material, verification methods, 
```

Comment:
I recommend also removing the sentence that a DID Document describes how to use a specific DID (which it doesn't do). A DID is just an identifier and a DID Document does not tell you how to use it. It does, however, express the information described in the next sentence.

```suggestion
Each <a>DID document</a> can express cryptographic material, verification methods, 
```

Reply:
@dlongley 
> I recommend also removing the sentence that a DID Document describes how to use a specific DID (which it doesn't do). A DID is just an identifier and a DID Document does not tell you how to use it. It does, however, express the information described in the next sentence. 

Thanks for the corrections. I committed both of them.